### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by will-moore**
See README at https://github.com/will-moore/omero-py/tree/readme_update

I wanted to make this a bit more user-friendly.

### CODE REVIEW

**Comment by jburel** [comment]
since you have a list of command to want to put ``export OMERODIR=....``

**Comment by jburel** [comment]
``Successfully installed Pillow-6.2.1 future-0.18.2 omero-py-5.6.dev9``
numpy is not installed

File: README.rst [hunk 1]
```diff
@@ -19,16 +19,40 @@ Dependencies
...
+    export OMERODIR=$(pwd)
+
+See: `OMERO`_ documentation for more details.
 
 Usage
 -----
 
-See: `OMERO`_ documentation
+- For OMERO python language bindings, see `OmeroPy`_.
+- For Command-line usage, see `Omero-CLI`_.
```
Comment (mtbc) [inline README.rst]:
```suggestion
- For Command Line usage, see `Omero-CLI`_.
```

File: README.rst [hunk 2]
```diff
@@ -19,16 +19,40 @@ Dependencies
...
 Installation
 ------------
 
-See: `OMERO`_ documentation
+We recommend creating a virtualenv::
+
+    python3.6 -mvenv venv
+    . venv/bin/activate
+    pip install omero-py
+
```
Comment (manics) [inline README.rst]:
```suggestion

You may need to replace ``python3.6`` with ``python`` or ``python3`` depending on your Python distribution.

```

File: README.rst [hunk 3]
```diff
@@ -19,16 +19,40 @@ Dependencies
...
 
 Installation
 ------------
 
-See: `OMERO`_ documentation
+We recommend creating a virtualenv::
+
+    python3.6 -mvenv venv
+    . venv/bin/activate
+    pip install omero-py
```
Comment (manics) [inline README.rst]: This attempts to install `5.5.1` which fails.

**Comment by mtbc** [comment]
No blockers from me.

**Pull request merged**